### PR TITLE
Sample app

### PR DESCRIPTION
**Pull request by ramisaban1**
@benjo0010 @maciejmiarecki

### CODE REVIEW

File: sample/src/main/java/com/robotemi/sdk/sample/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+
+    //Primarily meant for applications that run in the background
+    @Override
+    public void onPlayButtonClicked(boolean play) {
+        //Event listeners for the media bar on iHeartRadio
+    }
+
+    //Primarily meant for applications that run in the background
+    @Override
+    public void onNextButtonClicked() {
```
Comment (benjo0010) [inline sample/src/main/java/com/robotemi/sdk/sample/MainActivity.java]: remove listener - not relevant anymore

File: sample/src/main/java/com/robotemi/sdk/sample/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+        //Event listeners for the media bar on iHeartRadio
+    }
+
+    //Primarily meant for applications that run in the background
+    @Override
+    public void onBackButtonClicked() {
```
Comment (benjo0010) [inline sample/src/main/java/com/robotemi/sdk/sample/MainActivity.java]: remove listener - not relevant anymore

File: sample/src/main/java/com/robotemi/sdk/sample/MainActivity.java [hunk 3]
```diff
@@ -0,0 +1,313 @@
...
+        //Event listeners for the media bar on iHeartRadio
+    }
+
+    @Override
+    public void onTrackBarChanged(int position) {
```
Comment (benjo0010) [inline sample/src/main/java/com/robotemi/sdk/sample/MainActivity.java]: remove listener - not relevant anymore

File: sample/src/main/java/com/robotemi/sdk/sample/MainActivity.java [hunk 4]
```diff
@@ -0,0 +1,313 @@
...
+
+    @Override
+    public void onPublish(ActivityStreamPublishMessage message) {
+        //After the activity stream finished publishing (photo or otherwise).
+        //Do what you want based on the message returned.
+    }
+
+    //Primarily meant for applications that run in the background
+    @Override
+    public void onPlayButtonClicked(boolean play) {
```
Comment (benjo0010) [inline sample/src/main/java/com/robotemi/sdk/sample/MainActivity.java]: remove listener - not relevant anymore

File: sample/src/main/java/com/robotemi/sdk/sample/MainActivity.java [hunk 5]
```diff
@@ -0,0 +1,313 @@
...
+    @Override
+    public void onTtsStatusChanged(TtsRequest ttsRequest) {
+
+        // Do whatever you like upon the status changing. after the robot finishes speaking
+        // Toast.makeText(this, "speech: " + ttsRequest.getSpeech() + "\nstatus:" + ttsRequest.getStatus(), Toast.LENGTH_LONG).show();
+    }
+
+    @Override
+    public void onBeWithMeStatusChanged(String status) {
+      //  When status changes to "lock" you can snap a picture.
```
Comment (benjo0010) [inline sample/src/main/java/com/robotemi/sdk/sample/MainActivity.java]: this comment is for Kellogs write something about  it means the robot recognize a user and following after him

File: sample/src/main/java/com/robotemi/sdk/sample/MainActivity.java [hunk 6]
```diff
@@ -0,0 +1,313 @@
...
+        versionsDialog.setAdapter(customAdapter, null);
+        AlertDialog dialog = versionsDialog.create();
+        dialog.show();
+    }
+
+
+    // TODO: 2019-06-16 Work on onNlpCompleted. At the moment I couldn't get it to load.
+    @Override
+    public void onNlpCompleted(NlpResult nlpResult) {
+        //do something with nlp result. Base the action on what you get back
```
Comment (benjo0010) [inline sample/src/main/java/com/robotemi/sdk/sample/MainActivity.java]: Write something about those actions needs to be declared in the skill manifest

File: sample/src/main/java/com/robotemi/sdk/sample/MainActivity.java [hunk 7]
```diff
@@ -0,0 +1,313 @@
...
+        if (isReady) {
+            try {
+                final ActivityInfo activityInfo = getPackageManager().getActivityInfo(getComponentName(), PackageManager.GET_META_DATA);
+                robot.onStart(activityInfo);
+            } catch (PackageManager.NameNotFoundException e) {
+                throw new RuntimeException(e);
+            }
+        }
+    }
+
```
Comment (benjo0010) [inline sample/src/main/java/com/robotemi/sdk/sample/MainActivity.java]: remove empty line

File: sample/src/main/java/com/robotemi/sdk/sample/MainActivity.java [hunk 8]
```diff
@@ -0,0 +1,313 @@
...
+        robot.removeOnGoToLocationStatusChangedListener(this);
+        robot.removeConversationViewAttachesListenerListener(this);
+        robot.removeWakeupWordListener(this);
+        robot.removeTtsListener(this);
+        robot.removeOnLocationsUpdateListener(this);
+    }
+
+    /*
+        onRobotReady establishes if a connection to the robot's navigation is made
+        as well as places the this application in the top bar for a quick access shortcut.
```
Comment (benjo0010) [inline sample/src/main/java/com/robotemi/sdk/sample/MainActivity.java]: remove this line in the comment

File: sample/src/main/java/com/robotemi/sdk/sample/MainActivity.java (hunk 6 above)
Comment (ramisaban1) [reply sample/src/main/java/com/robotemi/sdk/sample/MainActivity.java]: I will write about what needs to be declared in the manifest, but what does the onNlpCompleted method have to do with it?

**Pull request merged**